mbf_abstract_nav/src/abstract_planner_execution.cpp

```diff
@@ -285,14 +296,15 @@ namespace mbf_abstract_nav
 
         // unlock goal
         goal_start_mtx_.unlock();
+        ROS_INFO_STREAM("goal_start_mtx_ unlocked in while planning_ loop.");
```

Comment:
we might want to move this up one line. this could result in confusing logs where a lock is acquired before we log that we're unlocking